docs/environment-docs.yml

```diff
@@ -1,12 +1,12 @@
-name: jupytergis-docs
+name: jupytergis
```

Comment:
I would prefer to keep the name as `jupytergis-docs` because I already have an environment titled `jupytergis` on my machine and prefer to keep it :)